src/PKSim.Core/Services/ParameterTask.cs

```diff
@@ -273,9 +285,9 @@ public ICommand SetParameterDisplayValueAsStructureChange(IParameter parameter,
          return SetParameterValueAsStructureChange(parameter, parameter.ConvertToBaseUnit(valueToSetInGuiUnit));
       }
 
-      public ICommand SetParameterValueAsStructureChange(IParameter parameter, double value)
+      public ICommand SetParameterValueAsStructureChange(IParameter parameter, double value, bool shouldUpdateDefaultStateAndValueOriginForDefaultParameter = true)
       {
-         return withUpdatedDefaultStateAndValue(new SetParameterValueStructureChangeCommand(parameter, value).Run(_executionContext), parameter);
+         return withUpdatedDefaultStateAndValue(new SetParameterValueStructureChangeCommand(parameter, value).Run(_executionContext), parameter, shouldUpdateDefaultStateAndValueOriginForDefaultParameter: shouldUpdateDefaultStateAndValueOriginForDefaultParameter);
       }
```

Comment:
`shouldUpdateDefaultStateAndValueOriginForDefaultParameter:shouldUpdateDefaultStateAndValueOriginForDefaultParameter);` 
named parameter not really required here

Reply:
No. There is another default parameter so name parameter is required